src/operator/numpy/random/np_bernoulli_op.cc

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2019 by Contributors
+ * \file np_bernoulli_op.cc
+ * \brief Operator for numpy sampling from bernoulli distributions
+ */
+
+#include "./np_bernoulli_op.h"
+#include "./dist_common.h"
+
+namespace mxnet {
+namespace op {
+
+DMLC_REGISTER_PARAMETER(NumpyBernoulliParam);
+
+NNVM_REGISTER_OP(_npi_bernoulli)
+.describe("Sample frmo bernoulli distribution")
```

Comment:
Typo: from

Reply:
Actually I think `describe` may not even be needed for `_npi_` operators cause the actual docs seen by the users are the docstrings for the Python wrappers.

Reply:
I see. So it would be safe to even remove this.

Reply:
Yes that's what I mean. @xidulu do you want to remove this line?

Reply:
I will just remove it.